app/packages/looker-3d/src/labels/cuboid.tsx

```diff
@@ -22,6 +30,7 @@ export const Cuboid = ({
   color,
   useLegacyCoordinates,
 }: CuboidProps) => {
+  const [lineWidth] = useRecoilState(cuboidLabelLineWidthAtom);
```

Comment:
nit: `useRecoilValue` can be used when read only
```suggestion
  const lineWidth = useRecoilValue(cuboidLabelLineWidthAtom);
```